CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 * [FEATURE] Add new thermal_zone collector #1425
 * [FEATURE] Add new cooling_device metrics to thermal zone collector #1445
 * [FEATURE] Add new softnet collector #1576
+* [FEATURE] Add unix socket support for supervisord collector #262
```

Comment:
This is an `[ENHANCEMENT]`.

Reply:
Updated in next commit push https://github.com/prometheus/node_exporter/pull/1592/commits/b81ec9ee1dd2e5bfa69df21930d540b5fae3674c